### PR TITLE
Fixing bug in regex for phone field

### PR DESCRIPTION
**Pull request by megoth**
The regex for phones is still not great, but we'll leave it at this for now until we have a more thorough change of the profile pane

### CODE REVIEW

File: src/widgets/index.js [hunk 1]
```diff
@@ -824,7 +824,7 @@ UI.widgets.fieldParams[UI.ns.ui('TextField').uri] = { }
 
 UI.widgets.fieldParams[UI.ns.ui('PhoneField').uri] = { 'size': 20, 'uriPrefix': 'tel:' }
 UI.widgets.fieldParams[UI.ns.ui('PhoneField').uri].pattern =
-  /^\s*\+?[ 0-9-]+[0-9]\s*$/
+  /^\s*\+?[0-9-]+[0-9]\s*$/
```
Comment (Vinnl) [inline src/widgets/index.js]: Shouldn't we also disallow whitespace on either side of the phone number?

**Comment by Vinnl** [comment]
I'm fine with merging, so go ahead if you want - just not sure if whitespace at the start or end of a phone number won't cause problems either :)

File: src/widgets/index.js (hunk 1 above)
Comment (RubenVerborgh) [reply src/widgets/index.js]: While we're at it, shall we just `\d`?

Comment (megoth) [reply src/widgets/index.js]: Can do - wanted to keep the changes minimal, as to not introduce any errors (I'm not very good with regex, after all). But I've tried a slight change now, hopefully to the better.

**Comment by megoth** [comment]
> I'm fine with merging, so go ahead if you want - just not sure if whitespace at the start or end of a phone number won't cause problems either :)

Tried to improve the regex used - please double-check, then we'll start publishing updates to NPM.

**Pull request merged**